pkgs/applications/misc/curaengine/default.nix

```
@@ -16,6 +16,11 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DCURA_ENGINE_VERSION=${version}" ];
 
+  CXXFLAGS = [
+    # TODO missing import needed for GCC 13 already added upstream, remove in next release
+    "-include cstdint"
+  ];
+
```

```suggestion
  env. CXXFLAGS = toString [
    # TODO missing import needed for GCC 13 already added upstream, remove in next release
    "-include cstdint"
  ];

```

env vars are moving under `env`. i'd prefer the upstream patch to including cstdint everywhere.

---

Didn't know about `env`, thanks!
I reverted the branch to the previous commit, I also was torn between having a patch (more difficult to maintain) or import that library everywhere (easier to maintain but uglier). But better thinking about it in this case the import problem is already fixed in `master` upstream, so the first solution is not less maintainable.